src/contexts/states/LogFileManagerStore.ts

```diff
@@ -0,0 +1,18 @@
+import * as Comlink from "comlink";
+import {create} from "zustand";
+
+import {WrappedLogFileManager} from "../../services/WrappedLogFileManager";
+
+
+interface LogFileManagerState {
+    wrappedLogFileManager: Comlink.Remote<WrappedLogFileManager>;
+}
```

Comment:
@hoophalab How about
```suggestion
interface LogFileManagerProxyState {
    logFileManagerProxy: Comlink.Remote<LogFileManagerProxy>;
}
```